### PR TITLE
docs(website): fix outdated sdk and avd manager directories

### PR DESCRIPTION
**Pull request by dyvapandhu**
## Description

- This pull request addresses the issue described here: #4255

In this pull request, I have change updated directory of sdk and avdmanager

---

> _For features/enhancements:_
 - [x] I have added/updated the relevant references in the [documentation](https://github.com/wix/Detox/tree/master/docs) files.

> _For API changes:_
 - [ ] I have made the necessary changes in the [types index](https://github.com/wix/Detox/blob/master/detox/index.d.ts) file.

### CODE REVIEW

File: docs/introduction/preparing-for-ci.md [hunk 1]
```diff
@@ -172,9 +172,10 @@ detox_e2e:
     - mkdir -p /root/.android && touch /root/.android/repositories.cfg
     # The Dockerimage provides two paths for sdkmanager and avdmanager, which the defaults are from $ANDROID_HOME/cmdline-tools
     # That is not compatible with the one that Detox is using ($ANDROID_HOME/tools/bin)
-    - echo yes | $ANDROID_HOME/tools/bin/sdkmanager --channel=0 --verbose "system-images;android-27;default;x86_64"
+    - echo yes | $ANDROID_HOME/cmdline-tools/latest/bin/sdkmanager --channel=0 --verbose "system-images;android-27;default;x86_64"
```
Comment (noomorph) [inline docs/introduction/preparing-for-ci.md:175]:
```suggestion
    - echo yes | $ANDROID_HOME/cmdline-tools/latest/bin/sdkmanager --channel=0 --verbose "system-images;android-27;default;x86_64" "emulator"
```

File: docs/introduction/preparing-for-ci.md [hunk 2]
```diff
@@ -172,9 +172,10 @@ detox_e2e:
...
+    - echo yes | $ANDROID_HOME/cmdline-tools/latest/bin/sdkmanager "emulator"
```
Comment (noomorph) [inline docs/introduction/preparing-for-ci.md:176]:
```suggestion
```

**Review by noomorph — CHANGES_REQUESTED** [review]
@dyvapandhu , maybe it's worth to unite these two lines, like in my suggestion?

File: docs/introduction/preparing-for-ci.md (hunk 1 above)
Comment (dyvapandhu) [reply docs/introduction/preparing-for-ci.md:175]: i see, thankyou for your suggestion. will fix ASAP

Comment (dyvapandhu) [reply docs/introduction/preparing-for-ci.md]: updated @noomorph

**Review by noomorph — APPROVED** [review]

**Pull request merged**